site/content/extend/webapp/hello-world.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Quick Start (Hello, world!)
+date: 2017-10-26T17:54:54-05:00
+subsection: webapp
+weight: -10
+---
+
+# Hello, world!
+
+This guide will walk you through the basics of extending the Mattermost web app.
+
+## Prerequisites
+
+The Mattermost web app is built using [ReactJS](https://reactjs.org/) with [Redux](https://redux.js.org/) and the plugins will use the same, so it's a good idea to become familiar with those first. You're also going to need [npm](https://www.npmjs.com/get-npm).
+
+Once you have npm installed, install the Mattermost Developer Kit (MDK) tool which will help you set up your workspace.
+
+```
+npm install -g mdk
+```
+
+You'll also need a Mattermost server to install and test the plugin. This server must have "Enable" and "EnableUploads" set to true in the "PluginSettings" section of its config file.
+
+## Setting up the Workspace
+
+To quickly set up a workspace, change into a directory where you want store your plugins in development and run:
+
+```
+mdk init plugin --name helloworld --components ProfilePopover --skip-prompts
+```
+
+This will create your plugin workspace in a directory called `helloworld`.
+
+In the future, you can run `mdk init plugin` to have prompts walk you through the setup.
+
+## Building the Plugin
+
+As you might have noticed with the command you ran above, this hello world example is going to override the ProfilePopover component of the Mattermost web app.
+
+Open up the popover component file at `./helloworld/webapp/components/profile_popover/profile_popover.jsx`.
+
+Scroll down to the `render` function and change it to this:
+
+```
```

Comment:
I think you probably need to specify a language here to get syntax highlighting.

Reply:
Yeah I had `javascript` there but it barfed on the JSX tags